### PR TITLE
Integration test for studio

### PR DESCRIPTION
**Pull request by akartsky**
Integration test for Domain, UserProfile and App resource 

Added only the test files in this PR so that it's easier to review.
Will rebase once everything is merged in. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ryansteakley — APPROVED** [review]

**Review by goswamig — APPROVED** [review]
LGTM

File: test/e2e/tests/test_studio.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration tests for the SageMaker ProcessingJob API.
```
Comment (mbaijal) [inline test/e2e/tests/test_studio.py:13]: Comment is incorrect.

**Review by mbaijal — APPROVED** [review]
Missing the update test. Also the adoptedResource should be added post release.

File: test/e2e/tests/test_studio.py (hunk 1 above)
Comment (akartsky) [reply test/e2e/tests/test_studio.py]: fixed

**Review by mbaijal — APPROVED** [review]

**Pull request merged**